---
title: "About"
permalink: "/about/"
layout: page
---


# Research Projects

<br/>

<img src="/assets/images/homepage-general/graphical-abstract.jpg" style="display: block; margin: auto;" />

### [Lipidomics of bacteria, and unsaturated fatty acid production in Streptcococci](./lipidomics-project/index.md)

A special current emphasis of research in The Baker Lab is using lipidomics to better understand the ways in which bacteria modify their cell membranes to adapt to their environment, and in some cases, cause disease. All cells, and many viruses, have membranes, which are composed of lipid bilayers.  The chemical properties of most membrane lipids render them notoriously difficult to study.  As a result, lipidomics is perhaps the least utilized major ‘omics discipline, and a relative deficiency exists in understanding the consequences of the lipidome in various contexts, despite certainty in its biological importance.  Bacterial cells all have at least one membrane, while Gram negative organisms have two.  Bacteria produce a diversity of lipids to the extent that many species can, in fact, be identified be their lipid profile alone (Abel *et al.*, 1963).  

The Lactobacillales order of bacteria contains some of the most important pathogens and commensal organisms of the human microbiota.  This includes the genera *Streptococcus*, *Enterococcus*, and *Lactobacillus*.  Previous research has shown that a diversity of Lactobacillales increase the proportion of unsaturated fatty acids in their cell membranes in response to various environmental stresses including acid stress and oxidative stress (Fozo *et al.*, 2004).  In the case of the caries pathogen, *S. mutans,* this shift to a membrane containing a greater percentage of unsaturated fatty acids was required to withstand further acid or oxidative stress, and crucially, cause disease in a rat model of dental caries (Fozo & Quivey, 2004, Fozo & Quivey, 2004, Fozo *et al.*, 2007).  Our current research project on this topic seeks to address several questions and knowledge gaps raised by these observations:  (1) although it is known how *S. mutans* and other Lactobacillales produce unsaturated fatty acids, it is not known how this system is regulated and controlled (i.e., how it is turned on when needed), (2) it is not known how the unsaturated fatty acids are protective, and (3) although a similar response appears to occur in all tested Lactobacillales, it is not known if it is protective and/or required for virulence in organisms other than *S. mutans* or other disease contexts.  This is a particularly important question, as Lactobacillales contains other devasting pathogens such as *Streptococcus pyogenes, Streptococcus pneumoniae, Enterococcus faecalis,* and *Enterococcus faecium,* all responsible for significant human morbidity and mortality.  The bacterial lipid biosynthesis pathway is quite different at the molecular level than its eukaryotic counterpart, therefore it presents attractive targets for the development of novel antibiotics (Radka & Rock, 2022).  Indeed, one of the few novel classes of antibiotics discovered and utilized in the past 30 years, triclosan, targeted the reductase step in bacterial fatty acid biosynthesis (distinct from the steps involved in unsaturated fatty acid biosynthesis) (Radka & Rock, 2022).  Beyond this specific study and application, the impact of the bacterial lipidome on bacterial physiology and pathogenesis more broadly is an understudied field, with advancements in mass spectrometry technologies opening the door to lipidomics studies with a level of resolution not possible previously.  Going forward, we envision the Baker Lab being at the forefront of bacterial lipidomics research.
<br/><br/>




### [RESISFORCE: partnering Norway, Brazil, India, Canada, and the USA to further excellence in education, research, and innovation in the study of biofilms and antibiotic research](./RESISFORCE-project/index.md)

<br/>


<img src="/assets/images/projects-RESISFORCE/IMG_3827.jpg" style="display: block; margin: auto;" />

<br/>

Resistance of pathogenic microbes to antibiotics is a growing worldwide concern, with global deaths due to antibiotic resistance predicted to overtake global deaths due to cancer and become the #1 cause of death worldwide by 2050 (Brown *et al.*, 2017).  Our RESISFORCE project, funded by the Research Council of Norway, partners research labs from the Norwegian Institute of Public Health, the University of Oslo, TATA Consultancy Services (based in Delhi, India), the University of Campinas (Piracicaba, Brazil), McGill University (Montreal, Canada), the University of Illinois at Chicago, the Forsyth Institute, and OHSU to engage diverse trainees, clinicians, and scientists in educational outreach regarding the accelerating antibiotic resistance crisis and antibiotic stewardship.  The Baker Lab has been an active partner in this project since 2019, and has facilitated RESISFORCE outreach symposia at dental conferences in Brussels, Belgium (CED-IADR 2021) and Marseille, France (PER-IADR 2022), as well as intensive symposia and hands-on workshops for trainees at the University of Oslo (2023) and the University of Campinas (2019, 2022).  Our team has also produced a massive online open course (MOOC), entitled “Exploring the Landscape of Antibiotic Resistance in Microbiomes”, available on FutureLearn.  This free online course enables interested clinicians, researchers, students, as well as members of the public, to discover how antibiotic resistance has emerged as one of the most urgent public health threats, explore how the study of antibiotic resistance genes helps us understand antibiotic resistance, and get hands-on experience examining data using the ResistoXplorer online tool (<http://www.resistoxplorer.no> ; itself produced as a collaboration initiated through the RESISFORCE project (Dhariwal *et al.*, 2021)).  RESISFORCE has also sponsored several international researcher exchanges between labs participating on the project, provided networking opportunities, and fostered fruitful research collaborations between the participating labs (Junges *et al.*, 2018, Junges *et al.*, 2019, Junges *et al.*, 2019, Ricomini Filho *et al.*, 2019, Salvadori *et al.*, 2019, Dhariwal *et al.*, 2021, Bajalan *et al.*, 2022, Dornelas-Figueira *et al.*, 2023, Junges *et al.*, 2023).  Since dentists account for approximately 10% of all antibiotic prescriptions, and antibiotic resistance and stewardship are frequently neglected topics in dental school curriculum, the unique dentistry-focused international outreach of this program is expected to be particularly impactful (Ramanathan *et al.*, 2023).  

The Baker Lab will also be hosting our 4-day annual RESISFORCE symposium here at OHSU on September 9-15 of 2024.  This symposium will include a day of formal presentations by faculty from the RESISFORCE project, OHSU faculty in related fields, and regional leaders in microbiology and antibiotic resistance research (Friday, September 13, 2024). The symposium will also feature 3 days of interactive workshops for dental/graduate student and postdoctoral-level trainees (September 10-12, 2024).  The workshops will include group problem-based learning sessions and presentations, working with other trainees from Brazil, Norway, the USA, and Canada, as well as a hands-on bioinformatics workshop.  The symposium will be a tremendous opportunity to learn about current research in antibiotic resistance and oral health, and to network and interact on an international level with oral health researchers.  Interested readers are invited to stay tuned for more information during the coming months.

<br/><br/>

### [Nanopore sequencing:  leveraging an emerging technology](./nanopore-project/index.md)

As stated above, the development of culture-independent analysis of microbiomes was enabled by so-called next-generation sequencing (Bennett, 2004, Margulies *et al.*, 2005, Bentley *et al.*, 2008, McKernan *et al.*, 2009).  Illumina was the most broadly used sequencing platform of 2010s, and revolutionized the life sciences by reducing the cost of sequencing by orders of magnitude, providing exceptionally accurate sequencing data, and increasing throughput. However, the latest generation of emerging sequencing technologies (i.e., “third generation sequencing”) is in in the process of disrupting biomedical research once more (Athanasopoulou *et al.*, 2021).  One particular technology that has gained considerable traction in the last several years is nanopore sequencing (Oxford Nanopore Technologies, Inc. [ONT]) (Jain *et al.*, 2015).  While Illumina sequencing-by-synthesis technology generates sequences that are generally 150 or 300 base pairs of DNA in length, ONT sequencing theoretically has no upper limit on the length of output sequence, with single reads of over 1 million base pairs of DNA routinely reported (Jain *et al.*, 2015).  Because most genomes contain repeat regions spanning much longer than 300 base pairs, the puzzle that is a given genome cannot typically be completed with Illumina short-read sequencing alone (i.e., because all these short reads match up nonspecifically to all their cognate sequences in the repeat regions of the chromosome).  The long reads produced by ONT sequencing much more easily span the repeat regions of genomes enabling significantly easier assembly of complete chromosomes.  However, until recently ONT technology was plagued with a considerably higher error rate than competing technologies, limiting its applicability (Amarasinghe *et al.*, 2020).  This error rate decreased in recent years, with a crucial inflection point being reached in 2022, when studies illustrated that the contemporary ONT instrumentation and software could produce genome assemblies with an error rate on par with Illumina sequencing (Sereika *et al.*, 2022).  

The Baker Lab was an early adapter of ONT technology, and in 2022 published the first protocols to obtain multiple complete bacterial genomes simultaneously directly from saliva using ONT sequencing (Baker, 2022).  Several of the genomes completed using these methods were the first complete genome for their given species (*Candidatus* Saccharibacteria* HMT-870, *Candidatus* Saccharibacteria* HMT-348, *Actinomyces graevenitzii*) (Baker, 2021, Baker, 2022, Baker, 2022).  Particularly intriguing were the genomes from the enigmatic Saccharibacteria family, which are essentially tiny (even by bacterial standards) parasitic bacteria that depend on larger host bacteria to survive (He *et al.*, 2015).  The novel complete genomes from our study illustrated that the G6 group of Saccharibacteria likely has a different lifestyle, and possibly host and host dependencies than the more well-understood G1 group of Saccharibacteria (Baker, 2021).  These differences likely extend to ecological and pathogenic roles as well, as Saccharibacteria appear to have a relationship to inflammation and periodontal disease (albeit poorly understood at this stage) (Chipashvili *et al.*, 2021).  The ability to obtain complete genomes directly from complex samples, such as saliva, will revolutionize microbiology research, as it was previously only possible to obtain complete genomes of species that were isolated and cultivated in the lab in a pure culture (i.e., only incomplete, draft genomes could be obtained from metagenomes using earlier sequencing technologies)(Athanasopoulou *et al.*, 2021).  Obtaining genomes that are both complete and accurate is of importance because they then allow accurate identification and quantification of the species of interest in microbiome samples, and enable accurate prediction of the metabolic capabilities, and therefore ecological and pathogenic roles of the species (Venter *et al.*, 2004, Naito *et al.*, 2016).  This data can further guide wet-lab research and help scientists design experiments, isolate, cultivate, and study species that were previously intractable (Cross *et al.*, 2019).  In addition to genomics and metagenomics, The Baker Lab pioneered use of the ONT sequencing platform for RNA sequencing of oral bacteria (Baker *et al.*, 2022).  RNA sequencing via ONT has several advantages as well.  Because ONT can sequence native DNA and RNA molecules (unlike most sequencing methods, which must first reverse transcribe the RNA to cDNA, and/or amplify the DNA or RNA with PCR), ONT sequencing can detect base modifications, such as methylation, as well as noncanonical bases such as inosine (Garalde *et al.*, 2018, Tourancheau *et al.*, 2021, Begik *et al.*, 2022, Nguyen *et al.*, 2022).  The ability to detect DNA and RNA modifications on a genome wide or transcriptome wide scale is a major advance and is likely produce entire new fields of microbiology research.  Furthermore, the long reads enable the detection of co-transcribed genes and RNA isoforms on a transcriptome wide scale (Garalde *et al.*, 2018, Grunberger *et al.*, 2022).  As none of the OHSU core facilities currently provide nanopore sequencing, The Baker Lab is eager to collaborate or consult on OHSU projects that could benefit from an ONT approach.

